docs/visual-basic/developing-apps/programming/toc.md

```diff
@@ -0,0 +1,85 @@
+# [Accessing Computer Resources](visual-basic/developing-apps/programming/computer-resources/index.md)
```

Comment:
I think this file has the 81 warnings.  The target starts with a relative path, and when you moved these entries to a new TOC, the location moved.  I'd try to change this:

``` 
# [Accessing Computer Resources](visual-basic/developing-apps/programming/computer-resources/index.md) 
```

to this:

``` 
# [Accessing Computer Resources](computer-resources/index.md) 
```

And make similar changes throughout this file.